src/main/java/com/onevizion/scmdb/SqlScriptExecutor.java

```diff
@@ -57,13 +58,13 @@ public class SqlScriptExecutor {
     @Autowired
     private DataSource pkgDataSource;
 
-    public int execute(SqlScript script) {
+    public int execute(SqlScript script, boolean isCompileSchemas) {
```

Comment:
add overridden method and use it in `DbManager`
```
public int execute(SqlScript script) {
        return execute(script, true);
    }
```

Reply:
Done